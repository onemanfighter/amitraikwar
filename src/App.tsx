import React from 'react';
import {
  AppRouterProvider,
  LocalizationProvider,
  ThemeProvider,
} from '@providers';
import { HelmetProvider } from 'react-helmet-async';
import { CursorProvider } from '@components';

function App() {
  return (
    <ThemeProvider>
      <React.StrictMode>
        <HelmetProvider>
          <CursorProvider>
            <LocalizationProvider>
              <AppRouterProvider />
            </LocalizationProvider>
          </CursorProvider>
        </HelmetProvider>
      </React.StrictMode>
    </ThemeProvider>
  );
}

export default App;
