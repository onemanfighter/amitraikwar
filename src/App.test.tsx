import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import App from './App';
import './localization/config';

test('renders learn react link', () => {
  const { container } = render(<App />);

  expect(container).toMatchSnapshot();
  // [Todo]: Fix this test
  // expect(screen.getByText('App made by Amit')).toBeDefined();
});
