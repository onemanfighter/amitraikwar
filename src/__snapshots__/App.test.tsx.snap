// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div>
  <script
    id="chakra-script"
  >
    !(function(){try{var a=function(c){var v="(prefers-color-scheme: dark)",h=window.matchMedia(v).matches?"dark":"light",r=c==="system"?h:c,o=document.documentElement,s=document.body,l="chakra-ui-light",d="chakra-ui-dark",i=r==="dark";return s.classList.add(i?d:l),s.classList.remove(i?l:d),o.style.colorScheme=r,o.dataset.theme=r,r},n=a,m="light",e="chakra-ui-color-mode",t=localStorage.getItem(e);t?a(t):localStorage.setItem(e,a(m))}catch(a){}})();
  </script>
  <div>
    App made by Amit
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
